### ISSUE TITLE
[Supporting the platform - ADF - Azure data factory] Controls are not visible after system settings to 400%.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3843]
**User Experience:**
User will find difficulty to interact with controls if that controls are not visible after resizing to 400% as user won't proceed further.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1
Screen Resolution: 1280*1024

**Pre-Requisite:**
- Set to 400% from System Display-> Advance system settings.
- Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate and select Add data factory control at left panel.
- Now, navigate to various edit field like Account, Subscription etc. on Add data factory window.
- Verify Add data factory window screen at 400%.

**Actual Result:**
Controls are not visible after system settings to 400% on Add data factory window screen and due to this, user is unable to interact with such control.

**Expected Result:**
Controls should be visible after system settings to 400% on Add data factory window screen so that user could easily interact with control to proceed further.

![MAS1 4 10_Azure data factory](https://user-images.githubusercontent.com/72871089/101137892-8061ed00-3635-11eb-8d78-67fdb7112705.png)

**Comment by JasonYeMSFT** [comment]
@Shalini3457 #3477 is about the same issue. Could you verify if it can be accessible by adjusting the in-app zoom level?

**Comment by fhljys** [comment]
@Shalini3457 can you verify?
